.vscode/settings.json

```diff
@@ -1,4 +1,25 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "cSpell.words": ["bitblt", "pngjs"],
+  "workbench.colorCustomizations": {
+    "activityBar.activeBackground": "#5ae75e",
+    "activityBar.background": "#5ae75e",
+    "activityBar.foreground": "#15202b",
+    "activityBar.inactiveForeground": "#15202b99",
+    "activityBarBadge.background": "#7774eb",
+    "activityBarBadge.foreground": "#15202b",
+    "commandCenter.border": "#15202b99",
+    "sash.hoverBorder": "#5ae75e",
+    "statusBar.background": "#2de132",
+    "statusBar.foreground": "#15202b",
+    "statusBarItem.hoverBackground": "#1bc020",
+    "statusBarItem.remoteBackground": "#2de132",
+    "statusBarItem.remoteForeground": "#15202b",
+    "titleBar.activeBackground": "#2de132",
+    "titleBar.activeForeground": "#15202b",
+    "titleBar.inactiveBackground": "#2de13299",
+    "titleBar.inactiveForeground": "#15202b99"
+  },
+  "peacock.color": "#2de132"
```

Comment:
I think that adding `.vscode/settings.json` to git is nice but then we can run into things like this. 

For my use case, I always customize my projects with a plugin called `peacock` and I need it to add config into settings.json.

Also maybe someone may not use format on save since we are checking for that in our pipe

I think the best way to work here is to add   `.vscode/settings.json` to git ignore

We can keep a  `vscode/settings.example.json` , `vscode/extensions.example.json` just like we use to do with `.env.example` and add some info on the readme

Reply:
I see. I will investigate what can we do about it. We might add it to the .gitignore file